packages/oid4vci-rest-client/src/types/IOID4VCIRestClient.ts

```diff
@@ -0,0 +1,23 @@
+import { IAgentContext, IPluginMethodMap } from '@veramo/core'
+import { CredentialOfferFormat, Grant } from '@sphereon/oid4vci-common'
+
+export interface IOID4VCIRestClient extends IPluginMethodMap {
+  vciClientCreateOfferUri(args: IVCIClientCreateOfferUriRequestArgs, context: IRequiredContext): Promise<IVCIClientCreateOfferUriResponse>
+}
+
+export interface IVCIClientCreateOfferUriRequestArgs {
+  grants: Grant
```

Comment:
property is called grants, but is a single object. should it be an array or called grant?

Reply:
it's an object with multiple keys for each grant